bikeshop_project/registration/migrations/0006_auto_20190620_0334.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is the same migration as https://github.com/BridgeCityBicycleCoop/workstand/pull/92/files/3dbe69723874e4778959545bf298b8c7eb2ea4de#diff-4cc176be3524646a52ccc48c1bd41820

I am going to merge the heroku branch first so we can do these subsequent PRs via a build pipeline with Heroku

Reply:
removed a commit with the migration.